File: front/src/app/core/models/dados-endereco.ts
export class DadosEndereco {
  public id!: number;
  public logradouro!: string;
  public complemento!: string;
  public numero!: string;
  public cep!: string;
  public bairro!: string;
  public cidade!: string;
  public estado!: string;
  public pais!: string;
  public ativo!: boolean;

  constructor(init?: Partial<DadosEndereco>) {
    Object.assign(this, init);
  }
}

File: back/src/main/java/web/api/br/formulario/models/DadosEndereco.java
package web.api.br.formulario.models;

import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;

@Setter
@Getter
@Entity
@Table(schema = "SISTEMA_INSTITUICAO", name = "DADOS_ENDERECO")
public class DadosEndereco {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "ID_ENDERECO")
    private Long id;

    @Column(name = "RUA")
    private String rua;

    @Column(name = "COMPLEMENTO")
    private String complemento;

    @Column(name = "NUMERO")
    private String numero;

    @Column(name = "CEP")
    private String cep;

    @Column(name = "BAIRRO")
    private String bairro;

    @Column(name = "CIDADE")
    private String cidade;

    @Column(name = "ESTADO")
    private String estado;

    @Column(name = "PAIS")
    private String pais;

    @Column(name = "ATIVO")
    private boolean ativo;
}
